### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by sunilvr**
https://deploy-preview-3--sunil-ecommerce.netlify.app/index.html

### CODE REVIEW

File: styles/login.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.card-vertical{
+    gap:0.4rem;
+    height:30rem;
+    padding:1rem;
+    margin: 2rem 2rem 30rem 0; 
+}
+
+.main-login .terms{
+    color:dodgerblue;
```
Comment (jalajcodes) [inline styles/login.css]: you can use a variable here

**Review by jalajcodes — COMMENTED** [review]
Looks good. Just one small comment

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: styles/login.css (hunk 1 above)
Comment (sunilvr) [reply styles/login.css]: yes Jalaj. Pushed the change.
Thank you very much for the review.

**Comment by sunilvr** [comment]
> Looks good to me!

Thanks for the review @ApurvaSawant11

**Pull request merged**